docs/network-iot/run-an-lns/buy-an-oui.mdx

```diff
@@ -39,52 +35,82 @@ Requires basic familiarity with Linux and Bash command-line.
 
 ## Pricing
 
-The OUI is purchased with [Data Credits (DC)](/iot/transaction-fees). Costs are subject to change,
-but currently the OUI itself costs US$100 worth of DC and each DevAddr cost an additional US$100 in
-DC.
+Currently the purchasing of OUIs must be done by contacting the Helium Foundation. In the future it
+will be self service.
 
 DevAddr are sold in sequential blocks between 8 and 65,536 and any power of two.
 
-You must purchase a slab when purchasing an OUI, therefore, the total minimum cost is US$900 worth
-of Data Credits: $100 for the OUI itself and $800 for eight DevAddr's.
+You must purchase a slab when purchasing an OUI, therefore, the total minimum cost is US$900 : $100
+for the OUI itself and $800 for eight DevAddr's.
 
-[The OUI purchase transaction itself](https://github.com/helium/proto/blob/master/src/blockchain_txn_oui_v1.proto)
-incurs a fee based as does any other blockchain transaction; generally this will be $0.35-1.00 worth
-of DCs based on how the fields get filled and change the transaction size.
+You will find instructions on what you need and who to contact at the end of the next section.
 
 ---
 
-### Purchase Transaction
+### Purchasing OUI and devAddr Slab
 
-:::info Current Process
+Before you can purchase an OUI and devAddr Slab, you need to do the following.
 
-Please send an email to [hello@helium.foundation](mailto:hello@helium.foundation) to purchase an
```

Comment:
that's the current process, they need to email Foundation, it's not directly done on chain. @kent-williams